internal/db/eval_history_test.go

```diff
@@ -70,8 +70,6 @@ func TestListEvaluationHistoryFilters(t *testing.T) {
 				require.Equal(t, es1, row.EvaluationID)
 				require.Equal(t, EntitiesRepository, row.EntityType)
 				require.Equal(t, repo1.ID, row.EntityID)
-				require.Equal(t, repo1.RepoOwner, row.RepoOwner.String)
-				require.Equal(t, repo1.RepoName, row.RepoName.String)
```

Comment:
why did we have to remove these?

Reply:
Because else we'd need to return the repo owner via a JOIN in the ListEvaluationHistory call. And the intent was to remove that. We should not rely on any entity-specific info.